### PR TITLE
[CDAP-8320][CDAP-8576] Log Viewer Bug Fixes

### PR DESCRIPTION
**Pull request by elfenheart**
JIRAs:
- https://issues.cask.co/browse/CDAP-8576
- https://issues.cask.co/browse/CDAP-8320

Fixes:
- Logs Filtering to come from backend
- Jitters when logs reached the end

### CODE REVIEW

File: cdap-ui/app/directives/log-viewer/log-viewer.js [hunk 1]
```diff
@@ -653,6 +685,7 @@ const link = (scope) => {
 };
 
 angular.module(PKG.name + '.commons')
+  .value('THROTTLE_MILLISECONDS', 250) // throttle infinite scroll
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/log-viewer/log-viewer.js:703]: Is this being used anywhere?

**Review by ajainarayanan — COMMENTED** [review]
Just a minor comment otherwise 👍 LGTM

File: cdap-ui/app/directives/log-viewer/log-viewer.js (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/directives/log-viewer/log-viewer.js:703]: It's being used internally by `ngInfiniteScroll`

**Comment by bdmogal** [comment]
LGTM

**Comment by elfenheart** [comment]
Build: http://builds.cask.co/browse/CDAP-DRC5874-4

**Pull request merged**